src/Presence/gmail.json

```
@@ -0,0 +1,126 @@
+      "description": "Displayed when viewing primary emails."
+   },
+   "gmail.categorySocial": {
+      "message": "Viewing the Social category",
+      "description": "Displayed when viewing social emails."
+   },
+   "gmail.categoryUpdates": {
+      "message": "Viewing the Updates category",
+      "description": "Displayed when viewing updates emails."
+   },
+   "gmail.categoryPromotions": {
+      "message": "Viewing the Promotions category",
+      "description": "Displayed when viewing promotions emails."
+   },
+   "gmail.categoryForum": {
+      "message": "Viewing the Forum category",
+      "description": "Displayed when viewing forum emails."
+   },
+   "gmail.inLabel": {
+      "message": "Viewing label",
+      "description": "Displayed when a user is browsing emails in certain label"
+   },
+   "gmail.generalSettings": {
+      "message": "Configuring General settings",
+      "description": "Displayed when the user is modifying their settings"
+   },
+   "gmail.labelSettings": {
+      "message": "Configuring Labels settings",
+      "description": "Displayed when the user is modifying their labels"
+   },
+   "gmail.inboxSettings": {
+      "message": "Configuring Inbox settings",
+      "description": "Displayed when the user is modifying their inbox settings"
+   },
+   "gmail.accountSetting": {
+      "message": "Configuring Account and Import settings",
+      "description": "Displayed when the user is modifying their account's settings"
+   },
+   "gmail.filterSettings": {
+      "message": "Configuring Filters settings",
+      "description": "Displayed when the user is modifying their filters"
+   },
+   "gmail.fwdandpopSettings": {
+      "message": "Configuring Forwarding and POP/IMAP settings",
+      "description": "Displayed when the user is modifying their IMAP and POP settings"
+   },
+   "gmail.chatSettings": {
+      "message": "Configuring Chat and Meet settings",
+      "description": "Displayed when the user is modifying their chat settings"
+   },
+   "gmail.addonsSettings": {
+      "message": "Configuring Add-ons",
+      "description": "Displayed when the user is modifying their add-ons"
+   },
+   "gmail.advancedSettings": {
+      "message": "Configuring Advanced settings",
+      "description": "Displayed when the user is modifying their advanced settings"
+   },
+   "gmail.offlineSettings": {
+      "message": "Configuring Offline settings",
+      "description": "Displayed when the user is modifying their offline settings"
+   },
+   "gmail.oldThemesSettings": {
+      "message": "Configuring Themes",
+      "description": "Displayed when the user is modifying their themes settings"
+   },
+   "gmail.lookingForEmail": {
+      "message": "Looking for an email",
+      "description": "Displayed when the user is looking for mail"
+   },
+   "gmail.viewingAnEmail": {
+      "message": "Viewing an email",
+      "description": "Displayed when the user is viewing mail"
```

Does this display any information about the email? (title, sender, etc)
```suggestion
      "description": "Displayed when the user is viewing an email"
```

---

no my highness

---

thank you servant